### PR TITLE
rebranding

### PR DESCRIPTION
**Pull request by DawoudSheraz**
### Description
This is part 2 of PROD-2270, focused on updating the brand and styling looks. Some significant changes have been done with the update of paragon and the addition of brand package. The list includes:

- Setting the max-width of container_fluid to 1440px. This width was reduced in paragon https://github.com/edx/paragon/pull/533 but the reduced width caused a lot of styling issues in support tools
- Use `variant` instead of class to render different button styles
- Update tests' snapshots

#### Main Page
image

#### User Search
image

#### Search Results
image

image

image

#### Disable User
image

#### Change Enrollment Form
image

#### Create Entitlement
image

#### Entitlement Details
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,8 +30,9 @@
     "url": "https://github.com/edx/frontend-app-support/issues"
   },
   "dependencies": {
+    "@edx/brand": "npm:@edx/brand-edx.org@^1.4.2",
     "@edx/frontend-platform": "^1.8.1",
-    "@edx/paragon": "7.2.1",
+    "@edx/paragon": "^12.8.0",
```
Comment (awaisdar001) [inline package.json:35]: 🎆

File: src/users/__snapshots__/UserPage.test.jsx.snap [hunk 2]
```diff
@@ -32,10 +32,9 @@ exports[`User Page snapshot matches correctly 1`] = `
         type="text"
       />
       <button
-        className="btn btn-primary"
-        onBlur={[Function]}
+        className="btn-primary btn btn-primary"
+        disabled={false}
```
Comment (awaisdar001) [inline src/users/__snapshots__/UserPage.test.jsx.snap:36]: why this `disabled={false}`is added?

**Review by awaisdar001 — APPROVED** [review]
The changes looks good.

File: src/users/__snapshots__/UserPage.test.jsx.snap (hunk 2 above)
Comment (DawoudSheraz) [reply src/users/__snapshots__/UserPage.test.jsx.snap:36]: I am not sure about this one. The snapshots were outdated after I changed the button variants. I ran the command to update them.

**Review by saadyousafarbi — APPROVED** [review]
:shipit:

**Pull request merged**